pkg/addresses/addresses.go

```diff
@@ -35,7 +35,7 @@ type Book struct {
 // Get the addresses for the test environment.
 func GetTestBook() *Book {
 	return &Book{
-		Application:         common.HexToAddress("0xb72c832dDeA10326143831F1E5F1646920C9c990"),
+		Application:         common.HexToAddress("0x2E663fe9aE92275242406A185AA4fC8174339D3E"),
```

Comment:
Please move all updates to the application address to a separate commit. If possible, that commit should also contain the update of the emulator SDK version.